src/components/common/TmBalance.vue

```diff
@@ -469,30 +481,6 @@ export default {
   padding-right: 2.5rem;
 }
 
-.tokens-div {
-  position: absolute;
-  right: 1.25rem;
-  top: 3.5rem;
-}
-
-.tokens-div > .col {
-  margin-right: 1rem;
-}
-
-.token-denom {
-  font-size: 12px;
-}
-
-.token-balance {
-  font-weight: bold;
-}
-
-.currency-selector.tm-form-group {
-  position: absolute;
-  right: 1.25rem;
-  top: -0.7rem;
-}
-
```

Comment:
Yep. Yesterday I was looking at all this CSS code and thinking "oh god, I need to clean all this up!"

Thanks for doing it @jbibla !

Reply:
😇